# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libxrandr/c++.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Fix for inclusion from C++ code, such as Qt.

  - Rene Rebe <rene@exactcode.de>

--- libXrandr-1.2.0/include/X11/extensions/Xrandr.h.vanilla	2007-02-22 14:47:07.990673621 +0100
+++ libXrandr-1.2.0/include/X11/extensions/Xrandr.h	2007-02-22 14:47:14.647052947 +0100
@@ -296,7 +296,7 @@
 int
 XRRGetOutputProperty (Display *dpy, RROutput output,
 		      Atom property, long offset, long length,
-		      Bool delete, Bool pending, Atom req_type, 
+		      Bool _delete, Bool pending, Atom req_type, 
 		      Atom *actual_type, int *actual_format,
 		      unsigned long *nitems, unsigned long *bytes_after,
 		      unsigned char **prop);
